src/components/HomePageHeader/styles.module.css

```diff
@@ -9,11 +9,24 @@
 }
 
 .hero {
-  background: url('/static/img/bg-pattern-dark.png');
+  /* background: url('/static/img/bg-pattern-dark.png'); */
```

Comment:
don't like it?
other proposal?

Reply:
Initially, I thought of leaving only the black background; it seemed to me a more modern and minimalist look. If, instead, we want to keep a background image, as an alternative, I would have something similar to this: 

![image](https://github.com/fastify/website/assets/98093812/3b91af8f-3a36-494c-b5c2-9855266b9416)

What do you think? Otherwise, it's perfectly fine to keep the current one.